cmd/kops/replace.go

```diff
@@ -40,7 +40,7 @@ func NewCmdReplace(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "replace -f FILENAME",
-		Short: "Replace a resource by filename or stdin",
+		Short: "Replace a resource by filename or stdin.",
```

Comment:
Replace a cluster... I need some word smithing and help on this one. Wth does this func do ;)